src/main/java/com/google/codeu/servlets/MapsServlet.java

```diff
@@ -0,0 +1,73 @@
+package com.google.codeu.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import java.io.IOException;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/**
+ * Returns Starbucks Locations a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}]
+ */
+
+@WebServlet("/starbucks-data")
+public class MapsServlet extends HttpServlet {
+  JsonArray starbucksLocationsArray;
+  
+  /**
+   *Method to parse through a comma serpearted string.
+   */
+
+  public SbLocation parseFile(String line) {
+    String[] cells = line.split(",");
+    String country = cells[0];
+    double lat = Double.parseDouble(cells[1]);
+    double lng = Double.parseDouble(cells[2]);
+
+    SbLocation loc = new SbLocation(country, lat, lng);
+    return loc;
+  }
+
+  @Override
+  public void init() {
+    starbucksLocationsArray = new JsonArray();
+    new SbLocation();
+    Gson gson = new Gson();
+    String line = "";
```

Comment:
`String line;` should work right? There is no need to construct an empty string here since a string is constructed below.